src/util/detect-bugs/unregistered-transition.js

```diff
@@ -1,18 +1,19 @@
 // Detect https://issues.chromium.org/issues/360159391
-let dummy = document.createElement("div");
-document.body.appendChild(dummy);
+let document = globalThis.document;
+let dummy = document?.createElement("div");
+document?.body.appendChild(dummy);
 let property = "--foo-" + Date.now();
-dummy.style.cssText = `${property}: 1; transition: ${property} 1ms step-start allow-discrete`;
+if (dummy) dummy.style.cssText = `${property}: 1; transition: ${property} 1ms step-start allow-discrete`;
 
 /**
  * Detect if the browser is affected by the unregistered transition bug.
  * @see https://issues.chromium.org/issues/360159391
  * @type {Promise<boolean>}
  */
 export default new Promise(resolve => {
-	requestAnimationFrame(() => {
+	globalThis.requestAnimationFrame?.(() => {
```

Comment:
If we have no DOM, we cannot do meaningful detection, so maybe we should just exit early instead of adding conditionals?

Reply:
Good point, done in https://github.com/LeaVerou/style-observer/pull/109/commits/22a0ba4f72d4dc0345a105b9aa2262ca8c061dd7